Content.Shared/CCVar/CCVars.cs

```diff
@@ -1396,7 +1396,7 @@ public sealed class CCVars : CVars
         /// Restricts IC character names to alphanumeric chars.
         /// </summary>
         public static readonly CVarDef<bool> RestrictedNames =
-            CVarDef.Create("ic.restricted_names", true, CVar.SERVER | CVar.REPLICATED);
+            CVarDef.Create("ic.restricted_names", false, CVar.SERVER | CVar.REPLICATED);
```

Comment:
please keep this true, and other servers can override to false on their own config.toml.  The basic game mode is designed for RP so this should be default on.